interface/repository/centre_repository.go

```diff
@@ -0,0 +1,18 @@
+package repository
+
+import (
+	"api-booking-time/domain/model"
+	"api-booking-time/usecase/repository"
+)
+
+type centreRepository struct {
+	centres *[]*model.Centre
+}
+
+func OpenCentreRepository(centres *[]*model.Centre) repository.CentreRepository {
```

Comment:
Public functions, methods, interfaces, and struct need a comment (single or multiline) just above it.

This to add a proper description of what that element do.

```suggestion
// OpenCentreRespository returns a CentreRepository instance for ...
func OpenCentreRepository(centres *[]*model.Centre) repository.CentreRepository {
```

Please, add a small description to all public elements.